index.js

```diff
@@ -143,12 +143,18 @@ exports.decorateTerms = (Terms, { React }) => {
           if (this.state.isLoading) return;
           if (!this.state.isRecording) {
             this.titleElement = document.querySelector('header');
-            recording.startRecording(this.state.canvases, this.fileName);
+            this.setState({ isRecording: true });
```

Comment:
Why do this instead of toggle `prevState`?

Reply:
yes because we only want to set isRecording to false when the callback in recording.startRecording executes. Thus, we can't simply toggle it.

Reply:
yes because we only want to set isRecording to false when the callback in recording.startRecording executes. Thus, we can't simply toggle it.